images/libvirtd/libvirtd.sh

```diff
@@ -6,16 +6,9 @@ fatal() { echo "FATAL: $@" >&2 ; exit 2 ; }
 [[ -f /host/var/run/libvirtd.pid ]] && fatal "libvirtd seems to be running on the host"
 brctl show | grep virbr >/dev/null && fatal "libvirtd bridges are present on the host"
```

Comment:
Was not changed here, but I think even if libvirt is down, the bridge will stay since a qemu process might use it. Right?